packages/ckeditor5-core/tsconfig.json

```diff
@@ -1,15 +1,3 @@
 {
-  "compilerOptions": {
```

Comment:
A relative path outside the package would require a strict directory structure while developing plugins. Maybe we could use the package name prefixed path same as we do for imports between packages?

Reply:
```json
{
  "extends": "ckeditor5/tsconfig.json"
}
```